### PR TITLE
Fix some broken html links

### PR DESCRIPTION
**Pull request by krOoze**

### CODE REVIEW

File: appendices/VK_EXT_image_drm_format_modifier.txt [hunk 1]
```diff
@@ -449,8 +449,7 @@ _requires_ the application to set pname:size to 0.
   . [[VK_EXT_image_drm_format_modifier-fn3]]
     https://www.khronos.org/registry/EGL/extensions/MESA/EGL_MESA_image_dma_buf_export.txt[`EGL_MESA_image_dma_buf_export`]
   . [[VK_EXT_image_drm_format_modifier-fn4]]
-    https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/include/uapi/drm/drm_mode.h?id=refs/tags/v4.10#n392[`struct
-    drm_mode_fb_cmd2`]
+    link:++https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/include/uapi/drm/drm_mode.h?id=refs/tags/v4.10#n392++[`struct drm_mode_fb_cmd2`]
```
Comment (oddhack) [inline appendices/VK_EXT_image_drm_format_modifier.txt]: What's the purpose of this change?

Comment (krOoze) [reply appendices/VK_EXT_image_drm_format_modifier.txt]: I guess nothing. Link checker I used didn't like something about that link, but this didn't help.

File: chapters/resources.txt [hunk 2]
```diff
@@ -526,10 +526,11 @@ include::{generated}/api/structs/VkBufferViewCreateInfo.txt[]
 ****
   * [[VUID-VkBufferViewCreateInfo-offset-00925]]
     pname:offset must: be less than the size of pname:buffer
-  * [[VUID-VkBufferViewCreateInfo-offset-02749]]
-    If the <<features-texelBufferAlignment,texelBufferAlignment>> feature is
-    not enabled, pname:offset must: be a multiple of
+ifndef::VK_EXT_texel_buffer_alignment[]
+  * [[VUID-VkBufferViewCreateInfo-offset-00926]]
```
Comment (krOoze) [inline chapters/resources.txt]: This is reverting to old VUID and old VU text for the non-extension version of the VU.

File: appendices/VK_KHR_sampler_ycbcr_conversion.txt [hunk 3]
```diff
@@ -10,7 +10,6 @@ include::{generated}/meta/VK_KHR_sampler_ycbcr_conversion.txt[]
     No known IP claims.
 *Interactions and External Dependencies*::
   - Promoted to Vulkan 1.1 Core
-  - This extension interacts with `<<VK_EXT_debug_report>>`
```
Comment (krOoze) [inline appendices/VK_KHR_sampler_ycbcr_conversion.txt]: This seems like a copy-paste error.
@fluppeteer Did you have any other extension in mind here?

**Comment by stonesthrow** [comment]
Sending this to @oddhack as its editing and markup.

File: chapters/memory.txt [hunk 4]
```diff
@@ -1925,7 +1925,7 @@ The size of the exported file may: be larger than the size requested by
 slink:VkMemoryAllocateInfo::allocationSize.
 If pname:handleType is ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_DMA_BUF_BIT_EXT,
 then the application can: query the file's actual size with
-link:man:lseek(2)[lseek(2)].
+link:http://man7.org/linux/man-pages/man2/lseek.2.html[lseek(2)].
```
Comment (cubanismo) [inline chapters/memory.txt]: This one seems somewhat questionable to me, since man pages vary by system, so linking directly to an arbitrary version seems a bit odd.  I'm not clear the intent here was actually to try to link to man pages explicitly, so much as reference the man pages using standard notation.

Comment (krOoze) [reply chapters/memory.txt]: Well, it is the least arbitrary version I can get. It is link directly from `kernel.org` man-pages project.

Intent here is to fix a link. Browser says "address not recognized" for `man:`.

**Comment by oddhack** [comment]
We agreed to take this today. The man page link issue can be opened separately later if needed - personally I think having *some* valid link is better than none and POSIX is pretty widespread now.

**Comment by krOoze** [comment]
rebased and fixed conflict in `VK_KHR_shader_clock.txt`

**Comment by krOoze** [comment]
PS: forgot about the man. To be removed in the next push

File: chapters/memory.txt (hunk 4 above)
Comment (krOoze) [reply chapters/memory.txt]: reverted
